packages/yew/src/dom_bundle/fragment.rs

```diff
@@ -1,15 +1,15 @@
 use std::collections::VecDeque;
 use std::ops::{Deref, DerefMut};
 
+use wasm_bindgen::JsCast;
 use web_sys::{Element, Node};
 
-use crate::dom_bundle::BSubtree;
-use crate::html::NodeRef;
+use super::{BSubtree, DomSlot};
 use crate::virtual_dom::Collectable;
 
 /// A Hydration Fragment
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
-pub struct Fragment(VecDeque<Node>);
+pub struct Fragment(VecDeque<Node>, Option<Node>);
```

Comment:
```suggestion
pub(crate) struct Fragment(VecDeque<Node>, Option<Node>);
```

I have checked the Yew docs of `v0.20`, this type is not publicly accessible, so we can make the type `pub(crate)` instead of making shift method `pub(crate)`.

Reply:
The module itself is not public and re-exports `Fragment` only as `pub(crate)`. Not sure why I changed it to `pub(crate) shift`, will change that back.